ui/app/insights/src/pages/page-view.ts

```diff
@@ -0,0 +1,366 @@
+                    } else {
+                        this._loadedDashboards = dashboards;
+                    }
+                })
+            }
+        }
+    }
+
+
+    /* -------------------------------- */
+
+    // FETCH RELATED FUNCTIONS
+
+    protected async fetchAllDashboards(realmName?: string, loginRedirect: boolean = false): Promise<Dashboard[]> {
+        if(!realmName) {
+            realmName = manager.displayRealm;
+        }
+        let promise = this.getPromise('dashboard/all');
+        if(promise == undefined) {
+            promise = this.registerPromise('dashboard/all', manager.rest.api.DashboardResource.getAllRealmDashboards(realmName), true, false);
+        }
+        try {
+            const response = await promise;
+            return response.data;
+        } catch (ex) {
+            if(isAxiosError(ex) && ex.response.status == 403 && loginRedirect) {
+                manager.login();
+            }
+            return undefined;
+        }
+    }
+
+    protected async fetchDashboard(id: string, loginRedirect: boolean = true): Promise<Dashboard | undefined> {
+        let promise = this.getPromise('dashboard/' + id);
+        if(promise == undefined) {
+            promise = this.registerPromise(('dashboard/' + id), manager.rest.api.DashboardResource.get(id), true, false);
+        }
+        try {
+            const response = await promise;
+            return response.data;
+        } catch (ex) {
+            if(isAxiosError(ex) && ex.response.status == 403 && loginRedirect) {
+                manager.login();
+            }
+            return undefined;
+        }
+    }
+
+
+    /* ---------------------------- */
+
+    protected render(): TemplateResult {
+        const realmConfig = this.realmConfigs ? this.realmConfigs[this._realm] : undefined;
+        const selected: Dashboard | undefined = this._loadedDashboards?.find((d) => d.id == this._selectedId);
+        let logoMobile = realmConfig?.logoMobile;
+        if(logoMobile == undefined) {
+            logoMobile = DefaultMobileLogo;
+        }
+        const pageStyles = {
+            display: 'flex',
+            flexDirection: 'column',
+            pointerEvents: this.dashboardMenu?.isDrawerOpen() ? 'none' : 'auto'
+        }
+        return html`
+            ${when(!this.viewDashboardOnly, () => {
+                const realms = this._loadedRealms?.map((r) => ({ name: r.name, displayName: r.displayName }));
+                return html`
+                    <dashboard-menu id="dashboard-menu" .dashboards="${this._loadedDashboards}" .realms="${realms}" .realm="${this._realm}"
+                                    .selectedId="${this._selectedId}" .userId="${this._userId}" .loading="${this.isLoading()}" .logoMobileSrc="${logoMobile}"
+                                    @realm="${(ev: CustomEvent) => this.changeRealm(ev.detail.value)}"
```

Comment:
`or-header` has a realm picker to be exact, with the fetching logic of the realms sitting in `or-app`.
The Insights app doesn't use `or-header`, since it uses a custom sidebar/menu. (with options such as switching realms)
The realm picker is also a tiny bit different, being a big button instead of a dropdown text like in the manager.

Since in `or-header` it was implemented with `getContentWithMenuTemplate()` and a simple list of realm strings,
I made a similar implementation for the Insights app.

Reply:
@DonWillems My concern here is that there are inconsistencies in our different UIs

Reply:
Discussed it with @DonWillems and @pierrekil earlier.
The content of the sidebar is quite different, and the Insights app has a kind of "mobile"-approach.
I get what you're coming from though.

Here are some visuals for context:

**Header in Manager app:**
![or-header-mei2023](https://github.com/openremote/openremote/assets/27913110/7a441322-06bb-40ea-a125-2f273cb621e5)

---

**Sidebar in Insights app:**
![insights-sidebar](https://github.com/openremote/openremote/assets/27913110/1dbc9e1b-257b-4fbc-a8a1-d44bbe83f9bd)

Reply:
How does it look when you use a desktop? Still have the pullout sidebar?

Reply:
Yes. Previously it was the similar to the list on the Insights page, but chose to focus on the UX on consoles.
But we thought it also works fine on desktop.

Would be in a seperate iteration if we want to change it.